MobiFlight/MidiBoard/MidiBoardDefinition.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+
+namespace MobiFlight
+{
+    public class MidiBoardDefinition
+    {
+        // From deserialization of config
+        public string InstanceName;
+
+        // From deserialization of config
+        public string DifferingOutputName;
+
+        // From deserialization of config
+        public int EncoderNeutral = 0;
```

Comment:
The schema said this is a position if I remember right. I suggest calling this `EncoderNeutralPosition` to be super clear on its use.

Reply:
Also there should be doc comments on each of the properties to match what was listed in the json file.

Reply:
Renaming to EncoderNeutralPosition - I like.

Regarding the doc comment topic in general. I'm in the camp of trying to not write any method or variable comments at all, even if they are public. In general I would argue if a comment is necessary, the variable name or method name is not clear enough. Or the method is too long and does too many things and therefore a good name cannot be found. In my opinion when I feel a comment is necessary I should think about a renaming or restructuring instead.

Nevertheless I'll check where I think additional comments really make sense.

Reply:
It's not a huge deal but I like adding doc comments to objects that are serialized/deserialized. As is there's no way to know on the code side which of these are optional, for example.

Reply:
Good point, being optional is a good reason for a comment.

Reply:
And once you comment one of them you might as well comment all of them 🤣

Reply:
jaja